frontend/server/src/Controllers/User.php

```diff
@@ -4605,6 +4606,68 @@ public static function apiRevokeAPIToken(\OmegaUp\Request $r) {
         \OmegaUp\DAO\APITokens::deleteByName($r->user->user_id, $name);
         return ['status' => 'ok'];
     }
+
+    /**
+     * @return array{entrypoint: string, templateProperties: array{payload: VerificationParentalTokenDetailsPayload, title: \OmegaUp\TranslationString}}
+     *
+     * @omegaup-request-param string $token
+     */
+    public static function getVerificationParentalTokenDetailsScript(
+        \OmegaUp\Request $r
+    ): array {
+        $r->ensureIdentity();
+
+        $token = $r->ensureOptionalString(
+            'parental_verification_token',
+            required: false,
+            validator: fn (string $token) => preg_match(
+                '/^[a-zA-Z0-9]{25}$/',
+                $token
+            ) === 1
+        );
```

Comment:
Now this param shouldn't be optional, so you can use the function `ensureString` instead:
```suggestion
        $token = $r->ensureString(
            'parental_verification_token',
            validator: fn (string $token) => preg_match(
                '/^[a-zA-Z0-9]{25}$/',
                $token
            ) === 1
        );
```